### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by TrySound**
Just read source code

### CODE REVIEW

File: dist/ripple.js [hunk 1]
```diff
@@ -1,51 +1,55 @@
...
+        options = $.extend({}, defaults, options);
 
-            easing: 'linear'
-        };
+        $('body').on(options.on, selector, Trigger);
 
-        self.defaults = $.extend({}, self.defaults, options);
 
-        var Trigger = function(e) {
+        function Trigger(e) {
```
Comment (jakiestfu) [inline dist/ripple.js:34]: Why was this changed?

File: dist/ripple.js [hunk 2]
```diff
@@ -1,51 +1,55 @@
...
+
+        easing: 'linear'
+    };
 
-        self.selector = selector;
-        self.defaults = {
-            debug: false,
-            on: 'mousedown',
+    $(function ($) {
+        $.ripple('[data-ripple-init]');
```
Comment (jakiestfu) [inline dist/ripple.js:24]: Again, I do not agree with the auto init idea... Why couldn't any user just do this in their own codebase if they want it auto initialized?

File: dist/ripple.js [hunk 3]
```diff
@@ -1,51 +1,55 @@
...
+    });
 
-            opacity: 0.4,
-            color: "auto",
-            multi: false,
+    $.ripple = function(selector, options) {
+
+        if(selectors.indexOf(selector) > -1) {
```
Comment (jakiestfu) [inline dist/ripple.js]: This doesn't actually do what you intend, or can be easily circumvented:

``` html
<a href="#" class="btn">Ripple</a>
```

``` javascript
$.ripple('a');
$.ripple('.btn');
```

File: dist/ripple.js (hunk 1 above)
Comment (TrySound) [inline dist/ripple.js:34]: This function attached to event before declaration.

File: dist/ripple.js (hunk 2 above)
Comment (TrySound) [inline dist/ripple.js:24]: But somebody need. It's cool to use plugin without any calls.
If any user do not need this, he will not use data-ripple-init attribute.

File: dist/ripple.js (hunk 1 above)
Comment (TrySound) [inline dist/ripple.js:34]: function bubbling or something like that. Don't remember.
Just good JS feature.)

File: src/ripple.js [hunk 4]
```diff
@@ -1,47 +1,45 @@
...
-            easing: 'linear'
-        };
+        options = $.extend({}, defaults, options);
 
-        self.defaults = $.extend({}, self.defaults, options);
+        $('body').on(options.on, selector, Trigger);
 
-        var Trigger = function(e) {
+
+        function Trigger(e) {
```
Comment (jakiestfu) [inline src/ripple.js:30]: It's called hoisting, but in this instance, var Trigger was defined before it was used so there is no difference in switching to function Trigger

Comment (TrySound) [inline src/ripple.js:30]: Event delegation before Trigger function

``` js
$('body').on(options.on, selector, Trigger);
```

Comment (TrySound) [inline src/ripple.js:30]: More comfortable code, I think.

Comment (TrySound) [inline src/ripple.js:30]: Thanks for info. Forgot english term.

**Comment by TrySound** [comment]
Does opera prefix need?
Last prefixed version is 12.0. Who uses it?
I know old opera is alive, but it's 12.1.

Remove?
